projects/services-ogc/assets/zkiViewer.test.context.ts

```diff
@@ -0,0 +1,566 @@
+import { IOwsContext } from '../src/lib/owc/types/owc-json';
```

Comment:
Can you please remove this context,  we cann not consider any context as public.

If there are new features to test we should place them in existing contexts.

Reply:
... almost done, need to pass some more unit tests; will take a little while ...

Reply:
Alright, files have been removed!